contracts/DelFiPrice.sol

```diff
@@ -43,47 +37,8 @@ contract DelFiPrice is OpenOracleView {
     /// @notice The mapping of medianized prices per symbol
     mapping(string => uint64) public prices;
 
-    /// @notice The binary representation for 'ETH' symbol , used for string comparison
-    bytes32 constant symbolEth = keccak256(abi.encodePacked("ETH"));
-
-    /// @notice The binary representation for 'USDC' symbol, used for string comparison
-    bytes32 constant symbolUsdc = keccak256(abi.encodePacked("USDC"));
-
-    /// @notice The binary representation for 'DAI' symbol, used for string comparison
-    bytes32 constant symbolDai = keccak256(abi.encodePacked("DAI"));
-
-    /// @notice The binary representation for 'REP' symbol, used for string comparison
-    bytes32 constant symbolRep = keccak256(abi.encodePacked("REP"));
-
-    /// @notice The binary representation for 'BTC' symbol, used for string comparison
-    bytes32 constant symbolWbtc = keccak256(abi.encodePacked("BTC"));
-
-    /// @notice The binary representation for 'BAT' symbol, used for string comparison
-    bytes32 constant symbolBat = keccak256(abi.encodePacked("BAT"));
-
-    /// @notice The binary representation for 'ZRX' symbol, used for string comparison
-    bytes32 constant symbolZrx = keccak256(abi.encodePacked("ZRX"));
-
-    /// @notice Address of the cEther contract
-    address public immutable cEthAddress;
-
-    /// @notice Address of the cUSDC contract
-    address public immutable cUsdcAddress;
-
-    /// @notice Address of the cDAI contract
-    address public immutable cDaiAddress;
-
-    /// @notice Address of the cREP contract
-    address public immutable cRepAddress;
-
-    /// @notice Address of the cWBTC contract
-    address public immutable cWbtcAddress;
-
-    /// @notice Address of the cBAT contract
-    address public immutable cBatAddress;
-
-    /// @notice Address of the cZRX contract
-    address public immutable cZrxAddress;
+    /// @notice The mapping of token symbols to CToken contracts addresses
+    mapping(string => address) tokens;
```

Comment:
Won't these be much gassier to read?

Reply:
yes, moving one with immutable constants.